nvflare/fuel/hci/client/cli.py

```diff
@@ -266,6 +266,11 @@ def _do_default(self, line):
             ent = entries[0]
             resp = ent.handler(args, self.api)
             self.print_resp(resp)
+            if "data" in resp:
+                for item in resp["data"]:
+                    item_type = item["type"]
+                    if item_type == "error" and "session timed out" in item["data"]:
```

Comment:
We should use the "return code" in the resp (not "data" ) to communicate this kind of error.
The return code should be defined as a string without spaces (e.g. session_timeout).
Please check with Kevin about usage of the "return code".